integration/tests/addons/addons_test.go

```diff
@@ -12,8 +12,6 @@ import (
 	. "github.com/weaveworks/eksctl/integration/runner"
 	"github.com/weaveworks/eksctl/integration/tests"
 	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
-	"github.com/weaveworks/eksctl/pkg/eks"
-	kubewrapper "github.com/weaveworks/eksctl/pkg/kubernetes"
```

Comment:
this is causing the linter to fail, not sure why this happened :thinking:

Reply:
I think this may have happened when I rebased it. It's fixed now so the linter should pass.